blockchain/fundamentals/ethereum-i/what-does-ethereum-do-differently.md

```diff
@@ -23,7 +27,37 @@ aspects:
 ---
 ## Content
 
+Other than being distributed and having a PoW algorithm initially, Ethereum does pretty much everything else *differently*.
+
+### Account
+
+Ethereum is using an **account model** aka balance model. You are probably familiar with the centralized version of this since it's employed by banks. The problem with a decentralized account model is its vulnerability to *replay attacks*: if you broadcast a signed transaction in which you pay Bob 0.5 ETH, Bob could re-broadcast the same transaction in order to receive 1 ETH in total. To prevent these attacks, Ethereum transactions have a *sequence number* (nonce).
+
+### Transactions
+
+Ethereum accepts more than transfer transactions. Given its capability of running code, Ethereum has special transactions used to interact with smart contracts: creation, calling, state querying. 
+
+Regardless of transaction type, 4 variables are required:
+- Gas price: higher gas price means a higher transaction fee and makes nodes more likely of including the transaction in the next block
+- Gas limit: depending on the resources used by the smart contract, the amount of gas used is determined; if the gas limit is excedeed (transaction costs more than you wanted to pay), the transaction is dropped but the transaction fee is not returned
+- Nonce: transaction sequence number
+- From: transaction signatory
```

Comment:
isn't this the owner's address?

Reply:
not really - the contract owner may be somebody else than the one who sends the transaction